download.md

```diff
@@ -4,9 +4,66 @@ title: Downloads
 permalink: /downloads/
 ---
 
-Once the first version of PELUX is released you will be able to find download
-links to the images, SDK and ADK, etc.
+# PELUX 1.0
```

Comment:
Maybe there should be a place with link to the SWF (published docs not sources) with that version in the bottom or somewhere?